core/trino-main/src/test/java/io/trino/operator/window/matcher/MatchAssert.java

```diff
@@ -53,29 +53,31 @@ private MatchAssert(MatchResult actual, Map<IrLabel, Integer> labelMapping)
         return () -> new MatchAssert(matcher.run(identityEvaluator(mappedInput)), labelMapping);
     }
 
+    @SuppressWarnings("UnusedReturnValue")
```

Comment:
what are you suppressing here?

Reply:
Right, so this is part of the custom fluent API to allow chaining assertion calls. But this is actually optional, and in this particular case calls to this method are always terminal. IDEA (not ErrorProne this time) still complains that the return value is ignored. So it's kind of related to the rest of the commit, but not really.

Reply:
We shouldn't create impression it's related to error-prone. Make into separate commit

I was patentntly ignoring this warning (blaming IDE), but maybe i should have not. Add some short comment.

Reply:
Also, `@SuppressWarnings` on a method suppresses warning for the whole method, so this could mask issues. Maybe we don't want to do that at all?

Reply:
Well, I think I will just remove the annotation. There will always be false positives we have to deal with...

Reply:
Or we can just use `@CanIgnoreReturnValue`! Turns out IDEA knows about it